src/pages/LogIn/LogIn.js

```diff
@@ -1,7 +1,54 @@
 import React from 'react';
+import img from "./image 1.png"
+import fb from "./Style.png"
 
 function LogIn() {
-  return <h1 className="mx-auto text-center text-7xl">Log In</h1>;
+  return (
+    <div className="mx-24 my-10  pl-3 ">
```

Comment:
suggest to add to line 7 the following:

mx-auto lg:max-w-7xl px-10 py-10

and remove pl-3  mx-24 my-10